docs/release-notes/8.16.asciidoc

```diff
@@ -0,0 +1,115 @@
+[[release-notes-header-8.16.0]]
+== 8.16
+
+[discrete]
+[[release-notes-8.16.0]]
+=== 8.16.0
+
+[discrete]
+[[known-issue-8.16.0]]
+==== Known issues
+
+// tag::known-issue-189676[]
+[discrete]
+.Tags appear in Elastic AI Assistant's responses
+[%collapsible]
+====
+*Details* +
+On August 1, 2024, it was discovered that Elastic AI Assistant's responses when using Bedrock Sonnet 3.5 may include `<antThinking>` tags, for example `<search_quality_reflection>` ({kibana-issue}189676[#189676]).
+
+====
+// end::known-issue-189676[]
+
+[discrete]
+[[breaking-changes-8.16.0]]
+==== Breaking changes
+
+* During shutdown, {kib} now waits for all the ongoing requests to complete according to the `server.shutdownTimeout` setting. During that period, the incoming socket is closed and any new incoming requests are rejected. Before this update, new incoming requests received a response with the status code 503 and body { "message": "{kib} is shutting down and not accepting new incoming requests" }.
+
+[discrete]
+[[features-8.16.0]]
+==== New features
+
+// * Introduces a new API route for listing Entity Store entities: `GET /api/entity_store/entities/list` ({kibana-pull}192806[#192806]).
+* Introduces Knowledge Base for Elastic AI Assistant, which allows you to specify information for AI Assistant to remember when responding to your queries ({kibana-pull}186566[#186566], {kibana-pull}192665[#192665]).
+* Enables detection rules to automatically execute system actions such as opening a case ({kibana-pull}183937[#183937]). 
+* Adds syntax validation for {esql} queries ({kibana-pull}189780[#189780]).
+* Provides a way to view {es} queries that run during rule execution. This option is provided for {esql} and EQL rules only ({kibana-pull}191107[#191107]).
+* Allows you to create and update a rule even when some data-related validation errors are present in the query field ({kibana-pull}191487[#191487]).
+* Introduces a new advanced setting `securitySolution:enableVisualizationsInFlyout`. When enabled, you can examine alerts and events in the **Visualize** tab, which provides a more detailed view of the event analyzer and Session View ({kibana-pull}194012[#194012], {kibana-pull}192531[#192531]).
+* Creates a new advanced setting `securitySolution:excludedDataTiersForRuleExecution` that allows you to exclude cold and frozen data from rule execution. This setting does not apply to {ml} rules ({kibana-pull}186908[#186908]). 
+* Introduces the ability to add notes to alerts and events, in addition to Timeline.
+//PR for notes feature is incoming.
```

Comment:
This still true? Just highlighting in case it's missed 😉

Reply:
Resolving since this has been addressed - thanks!